examples/README.md

```diff
@@ -0,0 +1,94 @@
+
+URL to the specific project follow the undermentioned convention:
+
+`<version_number>/<category>/<framework>/<example_path>`
+
+An example URL to the live example: https://handsontable.github.io/handsontable/8.1.0/docs/js/settings
+
+For more details see the [Folder structure](#folder-structure) section.
+
+### Preview on CodeSandbox
+
+To preview the arbitary code example on CodeSandbox you must get the link to the project folder in the Gihub repo. For example: if the URL to the project is https://github.com/handsontable/handsontable/tree/master/examples/8.0.0/docs/js/settings you can preview that project on CodeSandbox adding the "box" word right after the "github" and before the ".com". The URL to the sandboxed project would be https://githubbox.com/handsontable/handsontable/tree/master/examples/8.0.0/docs/js/settings.
+
+### Creating new examples
+
+Read the [Templates Guideline](./templates/README.md) for the detailed information about new code examples creation.
+
+### Editing existing examples
+
+Sometimes you want to edit existing code examples that live in the `/examples/<version_number>` directory.
+
+1. Edit code examples within the `/examples/<version_number>` directory.
+2. Commit and push the code examples to the repo.
+3. In Github Actions run a manual workflow with the `<version_number>` input of the Handsontable and Handsontable's wrapper version.
+4. The code examples will be deployed to the Github Pages and will be available under the same URL as they already were.
+
+### Development
+
+You can launch the `http-server` or any other server if you want to test and check code examples live.
+
+To see code examples in action run these commands:
+
+1. `npm run examples:build <version_number>` - will build all the code examples in the `/examples/<version_number>` directory and copy each code example production output to the `/examples/tmp/<version_number>`. The path to the code example in the `/examples/tmp` follows the [Folder structure](#folder-structure) convention.
+2. `npm run examples:start` - it will start the `http-server` right in the `/examples/tmp` on PORT `8080`. So the URL to the specific code example would be `http://localhost:8080/8.1.0/docs/angular/custom-context-menu/`. The URL follows the same convention as mentioned in the [Live on production](#live-on-production) section.
+
+#### How to run the arbitrary code example
+
+You may want to launch an arbitrary code example instead of a whole folder with all code examples for a specific version. To do that you will need to copy the code example production output to the `/examples/tmp` directory. An example:
+
+To launch only one code example which path is `/examples/8.1.0/docs/angular/custom-context-menu` you must copy the `dist` folder from this example and paste it to the `/examples/tmp/<your_temporary_folder_name>`. After done this, you can run `npm run examples:start` and see your code example under the URL `http://localhost:8080/<your_temporary_folder_name>/`
```

Comment:
I'm not sure about this since we can run a code example by `npm run start` within its directory